custodia/secrets.py

```diff
@@ -174,7 +174,11 @@ def _list(self, trail, request, response):
                 'Content-Type'] = 'application/json; charset=utf-8'
             response['output'] = msg.reply(keylist)
         except CSStoreError:
+            self.logger.error('List function: Internal server error')
```

Comment:
Looking at it I think these should be debug level errors, not error level.
The error itself is already well defined and these strings should not be normally cluttering the logs as the user of the API receives a very specific error that will leade straight to the correct conclusion if you read the code.
In debug mode it may make sense to print these messages to make it easier to debug issues from the server side when a client is opaque and communications are encrypted, but that debugging, not error reporting.